test/integration/instance.test.js

```diff
@@ -2100,6 +2100,32 @@ describe(Support.getTestDialectTeaser('Instance'), () => {
         });
       });
     });
+
+    if (dialect === 'postgres') {
```

Comment:
dialect.match(/^postgres/)

Reply:
@sushantdhiman Updated. Let me know if you want any other changes.